tox.ini

```diff
@@ -7,10 +7,12 @@ skipsdist = True
 deps =
   black
   yamllint
+  ansible-lint
 
 [testenv:linters]
 install_command = pip install {opts} {packages}
 commands =
   black -v --diff --check {toxinidir}
   yamllint -d relaxed .github/ meta/runtime.yml galaxy.yml 
+  ansible-lint -t fqcn -x yaml
```

Comment:
Yeah because I'm waiting for the PR in ansible-datadog to be merged first otherwise the test would fail as it is rn and I wanted to check that at least the fqcn part was working as intended.

Reply:
I'm going to swap back to ``ansible-lint`` once the lint version upgrade PR is merged

Reply:
Oh right, I forgot :facepalm: